### PR TITLE
chore(ci): fix missing percy snapshots

### LINKED ISSUE #16232: fix missing percy snapshots after upgrade

### PR DESCRIPTION
**Pull request by kuceb**
- Closes #16232 

- fix removing leftover percy version
- migrate old percy configs
- use express as testing webserver for desktopgui/reporter cypress tests instead of `cp.exec('http-server')`, which was causing issues with percy asset capture
- (unrelated) rebalance some e2e specs for faster ci runs
- make cy.then use defaultCommandTimeout instead of cy.timeout as default timeout, which is in line with other commands and allows defaultCommandTimeout to be changed during test execution and take effect
### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has the original issue or this PR been tagged with a release in ZenHub?

### CODE REVIEW

**Review by jennifer-shehane — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -169,7 +169,6 @@
     "odiff-bin": "2.1.0",
     "parse-github-repo-url": "1.4.1",
     "patch-package": "6.2.2",
-    "percy": "^0.26.9",
```
Comment (flotwig) [inline package.json:172]: ?

Comment (flotwig) [reply package.json:172]: why is this deleted?

Comment (Bkucera) [reply package.json:172]: percy moved from percy to @percy/cli

Comment (Bkucera) [reply package.json:172]: added cli in prvious PR, forgot to remove this

**Review by flotwig — APPROVED** [review]

**Pull request merged**